### PR TITLE
Added ACCOUNT_FORCE_REDIRECT_IF_LOGGED_IN option

### PR DESCRIPTION
**Pull request by LuanP**
Allows choosing if you want logged in users to be redirected in login/signup pages.

### CODE REVIEW

File: allauth/account/app_settings.py [hunk 1]
```diff
@@ -183,6 +183,10 @@ def CONFIRM_EMAIL_ON_GET(self):
         return self._setting('CONFIRM_EMAIL_ON_GET', False)
 
     @property
+    def FORCE_REDIRECT_IF_LOGGED_IN(self):
```
Comment (pennersr) [inline allauth/account/app_settings.py]: Hmmm, what's in a name...  how about `AUTHENTICATED_LOGIN_REDIRECTS` ?

File: allauth/account/views.py [hunk 2]
```diff
@@ -57,7 +57,8 @@ def dispatch(self, request, *args, **kwargs):
         # WORKAROUND: https://code.djangoproject.com/ticket/19316
         self.request = request
         # (end WORKAROUND)
-        if request.user.is_authenticated():
+        if request.user.is_authenticated() and \
+                app_settings.FORCE_REDIRECT_IF_LOGGED_IN is True:
```
Comment (pennersr) [inline allauth/account/views.py]: No need for `is True`... simply leave that out.

File: docs/configuration.rst [hunk 3]
```diff
@@ -133,6 +133,13 @@ ACCOUNT_SESSION_COOKIE_AGE (=1814400)
   How long before the session cookie expires in seconds.  Defaults to 1814400 seconds,
   or 3 weeks.
 
+ACCOUNT_FORCE_REDIRECT_IF_LOGGED_IN( (=True)
+  The default behaviour is to redirect logged in users to
```
Comment (pennersr) [inline docs/configuration.rst]: ... to redirect authenticated users ...

**Comment by luanpablo-luizalabs** [comment]
@pennersr Hi, just updated it. Can you check it again?

Thanks.

**Comment by LuanP** [comment]
@pennersr Hi, just updated it. Can you check it again?

Thanks.

**Pull request merged**